WcaOnRails/app/webpacker/lib/hooks/useLoggedInUserPermissions.js

```diff
@@ -14,6 +15,19 @@ export default function useLoggedInUserPermissions() {
 
   const loggedInUserPermissions = React.useMemo(() => ({
     canViewDelegateAdminPage: () => Boolean(data?.can_view_delegate_admin_page.scope === '*'),
+    canEditRole: (_role) => {
```

Comment:
What is the underscore doing here?

Reply:
Oops, actually there was a `role` outside this function, but it got cleaned up during the course of rebasing 😅 Sorry for it, removed the `_` now.